### ISSUE TITLE
Reorganization of webpage code

### ISSUE THREAD

**Issue by bsoyka** [abaies/abaies.github.io#1]
The webpage should look almost exactly the same after this.

The goal of this is simply to make the code more readable. Arrange the HTML tags in whatever way is best, fix indentation issues, combine tags, etc.

**Comment by Nymrinae** [comment]
Hello, I can take care of this (and probably also #2 and #3), is it still needed ?

**Comment by bsoyka** [comment]
@Nymrinae Yes, it can still be done. I'll give these issues over to you, thanks!
